src/TableRow.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+function TableRow(props) {
+  const [highlight, setHighlight] = useState(false);
+
+  function highlightRow() {
+    setHighlight(!highlight);
+  }
+
+  function handleShowProfile(event) {
+    props.selectCustomer(props.result.id);
+    event.stopPropagation();
+  }
+
+  return (
+    <tr className={highlight ? "highlighted" : " "} onClick={highlightRow}>
+      <td>{props.result.id}</td>
+      <td>{props.result.title}</td>
+      <td>{props.result.firstName}</td>
+      <td>{props.result.surname}</td>
+      <td>{props.result.email}</td>
+      <td>{props.result.roomId}</td>
+      <td>{props.result.checkInDate}</td>
+      <td>{props.result.checkOutDate}</td>
+      <td>
+        {moment(props.checkOutDate).diff(moment(props.checkInDate), "days")}
```

Comment:
The number of days is not correctly showing on your website. There's a bug in line 27. Can you see what's causing your comparison to fail?